templates/partials/usages.html

```diff
@@ -0,0 +1,34 @@
+<h2>What can you build?</h2>
+<dl class="features">
+  <div>
+    <dt>
+      <div>
+        {{ icon_macros::sprite_icon(name="gears") }}
+        GUI Applications
+      </div>
+    </dt>
+    <dd>Perfect integration of GTK, Libadwaita, Libgranite makes development super simple. Publish easily your apps on Flathub without bloated packages.</dd>
+    <div>
+      <a href="https://developer.gnome.org/documentation/tutorials/beginners/getting_started.html">GNOME getting started tutorial</a>
+      <a href="https://docs.elementary.io/develop/writing-apps/hello-world">Elementary OS app tutorial</a>
+    </div>
+  </div>
+  <div>
+    <dt>
+      <div>
+        {{ icon_macros::sprite_icon(name="rocket") }}
+        Command Line Programs
+      </div>
+    </dt>
+    <dd>Use a large standard library and use loads of other libraries available for use in Vala. Vala makes also packaging and integration into operating systems easy.</dd>
+  </div>
+  <div>
+    <dt>
+      <div>
+        {{ icon_macros::sprite_icon(name="rocket") }}
+        Libraries
+      </div>
+    </dt>
+    <dd>Provide bindings from many other languages and API documentation without additional work. Also dynamic and static linking is both supported.</dd>
```

Comment:
> Provide bindings from many other languages and API documentation without additional work

It would be great if you could clarify what "without additional work" means.

Reply:
additional work as in crafting the bindings manually. It is all generated automatically to GIR.